### PR TITLE
fix: set sp_startletter field only with content

### PR DESCRIPTION
**Pull request by sitepark-hying**

### CODE REVIEW

File: src/Service/Indexer/SiteKit/DefaultSchema2xDocumentEnricher.php [hunk 1]
```diff
@@ -133,7 +133,9 @@ public function enrichDocument(
             ?: $base->getString('title');
         $doc->sp_sortvalue = $sortHeadline;
 
-        $doc->sp_startletter = $base->getString('startletter');
+        if ($base->has('startletter')) {
```
Comment (sitepark-veltrup) [inline src/Service/Indexer/SiteKit/DefaultSchema2xDocumentEnricher.php:136]: Why is this necessary, `getString()` returns an empty string if the field does not exist.

Comment (sitepark-hying) [reply src/Service/Indexer/SiteKit/DefaultSchema2xDocumentEnricher.php:136]: Bacause the solr Fiter 'sp_startletter:*' doesnt work
solr

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,4 @@
 /var/test/*
 /tools
 .phpactor.json
+target/
```
Comment (sitepark-veltrup) [inline .gitignore]: This is a maven directory. That shouldn't play a role here.

**Review by sitepark-veltrup — APPROVED** [review]
